### DISCUSSION TITLE
Question about level of bug reporting

### DISCUSSION THREAD

**Discussion by sivolc73** [bluecherrydvr/unity#290] [General]
Hello!
First of all thank you for this beautiful project, I'm super happy with the interface and where it's going, the server app has been solid and surprisingly lightweight, my server ended up way overkill!

I wanted to know if at this stage of development of the Unity app, what kind of bug hunting would you like? I don't want to drown @bdlukaa in small wrinkles and waste your development time in edge cases of non functionality breaking bugs if the app is not there yet.

I use the app at work, at home and on my phone so I encounter smaller bugs from time to time that seems repeatable and I'm wondering if I should report them right away or see if they survive for few versions.

Thanks!

**Comment by bdlukaa** [comment]
We are glad you like our products! Much effort has been put into then.

Please report any bugs you find. It helps us make our app stable enough for all users. 

We are close to release the stable version of the client, so the less the bugs the better.

Thanks.
